src/modules/displayComments.js

```diff
@@ -0,0 +1,40 @@
+const displayComments = async () => {
+  const commentId = document.querySelector('.movie-title').id;
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IR7MovTrVQtBQVyC6UTK/comments?item_id=${commentId}`;
+
+  const commentsList = document.querySelector('.movie-comments');
+  commentsList.innerHTML = '';
+
+  try {
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const commentsCounter = document.querySelector('.comments-counter');
+
+    if (response.error === 400) {
+      commentsCounter.innerHTML = '<p>Comments (0)</p>';
+    }
+
+    const commentsData = await response.json();
+
+    commentsCounter.innerHTML = `<p>Comments (${commentsData.length || 0})</p>`;
```

Comment:
According to the requirements, the counter functionality should be in a separate function in a different module and not depend on the data fetched from the API. The function should produce the number of comments depending on the data fetched from the DOM.👇👇

![noel review](https://github.com/NoelLincoln/movie-safari/assets/43038485/871c067e-d339-4171-ad59-d14aa9eff702)
Kindly fix that.